packages/blocks/src/api/i18n-block.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@gwwar, should we put here `variations` as well which we plan to integrate with `block.json` soon?

There is one more place where we would need to put it on the allow list.

Reply:
I think it'd be worth a try, will it affect the needed PR order?

Reply:
Let me think about what we need to change to enable `variations` in `block.json` on the PHP side. As far as I can tell, it's mostly the same set of changes like here now that `WP_Block_Type` and the REST API endpoint support block variations. Given that you won't be able to register block variations on the server until WP 5.8 is out the order shouldn't matter at all. We also have a mechanism in place that favors definitions from the server but fallback to the client when not set there. I think it's safe to change here at the same time 👍🏻

Reply:
Added in https://github.com/WordPress/gutenberg/pull/30293/commits/b851c2f3e3e91e8d200ba1036e3d7c6f17539d3a. Every change to docs get synced with https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/ so I will open another PR that covers the part that describes `variations` in the `block.json` file. We should plan to merge it around WordPress 5.8 RC.1 when all Dev Notes get published.